lib/sorcery/controller/submodules/remember_me.rb

```diff
@@ -10,9 +10,9 @@ def self.included(base)
           base.send(:include, InstanceMethods)
           Config.module_eval do
             class << self
-              attr_accessor :remember_me_httponly
+              attr_accessor :remember_me_httponly, :remember_me_secure
               def merge_remember_me_defaults!
-                @defaults.merge!(:@remember_me_httponly => true)
+                @defaults.merge!(:@remember_me_httponly => true, :@remember_me_secure => false)
```

Comment:
`nil` would likely be a safer default here. Still would need to sanity check how this interacts with `force_ssl` either way.